src/Sylius/Bundle/UserBundle/Resources/views/Customer/showProfile.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends 'SyliusUserBundle::layout.html.twig' %}
+
+{% block content %}
+<h4>{{ 'sylius.ui.welcome'|trans }}</h4>
```

Comment:
I'd remove stuff like that from default templates. We do not want to maintain that etc. We should only put stuff that is exactly minimum to get things working. Looking pretty is not the point, everyone has to customize it anyway and I bet not many shops will have big welcome text.